ipn_main_handler.php

```diff
@@ -470,6 +470,13 @@
       if ((int)$new_status == 0) $new_status = 1;
       if (in_array($_POST['payment_status'], array('Refunded', 'Reversed', 'Denied', 'Failed'))
            || substr($txn_type,0,8) == 'cleared-' || $txn_type=='echeck-cleared' || $txn_type == 'express-checkout-cleared') {
+        $sql = ("select orders_status from " . TABLE_ORDERS . "
+                 where orders_id = :ordersID:");
```

Comment:
I'd remove the open and close `(` parentheses `)` from this string, since we don't want the string to be treated as a boolean `true/false`.